From 51b8098dec69bec79924a58985e64d5ecee645b1 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:08:54 -0400
Subject: [powerpc] eeh: Fixup the brown paperbag fallout of the "cleanup"

Message-id: <1410545655-205645-306-git-send-email-dzickus@redhat.com>
Patchwork-id: 94239
O-Subject: [RHEL7 PATCH 305/626] powerpc: eeh: Fixup the brown paperbag fallout of the "cleanup"
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 57310c3c99eb6fab2ecbd63aa3f7c323341ca77e
Author: Thomas Gleixner <tglx@linutronix.de>
Date:   Wed Mar 5 00:06:11 2014 +0100

    powerpc: eeh: Fixup the brown paperbag fallout of the "cleanup"

    Commit b8a9a11b9 (powerpc: eeh: Kill another abuse of irq_desc) is
    missing some brackets .....

    It's not a good idea to write patches in grumpy mode and then forget
    to at least compile test them or rely on the few eyeballs discussing
    that patch to spot it.....

    Reported-by: fengguang.wu@intel.com
    Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
    Cc: Peter Zijlstra <peterz@infradead.org>
    Cc: Gavin Shan <shangw@linux.vnet.ibm.com>
    Cc: Benjamin Herrenschmidt <benh@kernel.crashing.org>
    Cc: ppc <linuxppc-dev@lists.ozlabs.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/kernel/eeh_driver.c b/arch/powerpc/kernel/eeh_driver.c
index 3e1d7de..bb61ca5 100644
--- a/arch/powerpc/kernel/eeh_driver.c
+++ b/arch/powerpc/kernel/eeh_driver.c
@@ -166,8 +166,9 @@ static void eeh_enable_irq(struct pci_dev *dev)
    *
    * tglx
    */
-  if (irqd_irq_disabled(irq_get_irq_data(dev->irq))
+  if (irqd_irq_disabled(irq_get_irq_data(dev->irq)))
    enable_irq(dev->irq);
+ }
 }
 
 /**
-- 
1.7.1